### PR TITLE
update dependencies and major fixes

### PR DESCRIPTION
**Pull request by joostoudeman**
Update the dependency versions. just to keep up.
Also remove the old xml support and other improvements

### CODE REVIEW

**Review by mxdrenthe — COMMENTED** [review]
Did you noted somewhere the jump in version from 1.x -> 2.x and why?

File: mavenplugin/src/test/projects/pom-to-test-yaml-2.xml [hunk 1]
```diff
@@ -43,7 +43,6 @@
         <configuration>
           <defaultContentRoot>registry</defaultContentRoot>
           <targetDir>${project.build.outputDirectory}2</targetDir>
```
[the remark this answers is not in the document]
Comment (joostoudeman) [reply mavenplugin/src/test/projects/pom-to-test-yaml-2.xml:45]: yes

File: bootstrapgenerator/src/main/java/nl/openweb/hippo/groovy/Generator.java [hunk 2]
```diff
@@ -68,11 +68,11 @@ protected Generator() {
 
     public static List<String> getAnnotations(final String script){
         //Strip comments
-        String codeBlock = script.replaceAll("\\s\\/\\*[\\w\\W]*\\*\\/", StringUtils.EMPTY);
+        String codeBlock = script.replaceAll("\\s/\\*[\\w\\W]*\\*/", StringUtils.EMPTY);
```
[the remark this answers is not in the document]
Comment (joostoudeman) [reply bootstrapgenerator/src/main/java/nl/openweb/hippo/groovy/Generator.java:71]: This one does not have the *

[the remark this answers is not in the document]
Comment (mxdrenthe) [reply bootstrapgenerator/src/main/java/nl/openweb/hippo/groovy/Generator.java:71]: A check

File: sampleproject/src/main/scripts/reloadcnd.groovy [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+import javax.jcr.Session
+
+@Bootstrap(contentroot = Bootstrap.ContentRoot.REGISTRY)
+@Updater(
+        name = "Update cnd's'",
+        description = '''This is a workaround script to reload cnd's without checks. 
+Ideal to remove doctypes from cnd's or change inheritence structures ''',
+        parameters = "cndreload.json"
+)
+class CndReloader extends BaseNodeUpdateVisitor {
```
[the remark this answers is not in the document]
Comment (mxdrenthe) [reply sampleproject/src/main/scripts/reloadcnd.groovy:28]: Let's keep it

**Review by mxdrenthe — APPROVED** [review]
LGTM

**Pull request merged**